data/scheduled/scheduled.go

```diff
@@ -5,29 +5,30 @@ import (
 	"github.com/ElrondNetwork/elrond-go-core/data"
 	"github.com/ElrondNetwork/elrond-go-core/data/block"
 	"github.com/ElrondNetwork/elrond-go-core/data/smartContractResult"
+	"github.com/ElrondNetwork/elrond-go-core/data/transaction"
 )
 
 // GetTransactionHandlersMap returns the smart contract results as a map of transaction handlers
 func (sscr *ScheduledSCRs) GetTransactionHandlersMap() map[block.Type][]data.TransactionHandler {
 	if sscr == nil {
 		return nil
 	}
-	if len(sscr.Scrs) == 0 {
+	if len(sscr.Scrs) == 0 && len(sscr.InvalidTransactions) == 0 {
 		return nil
 	}
 
 	result := make(map[block.Type][]data.TransactionHandler)
-	for i, scrs := range sscr.Scrs {
-		if len(scrs.TxHandlers) == 0 {
-			result[block.Type(i)] = nil
-			continue
-		}
-		transactionHandlers := make([]data.TransactionHandler, len(scrs.TxHandlers))
-		for j := range scrs.TxHandlers {
-			transactionHandlers[j] = scrs.TxHandlers[j]
-		}
-		result[block.Type(i)] = transactionHandlers
+	var smartContractResults []data.TransactionHandler
```

Comment:
Not needed make? Here and L27, L47 and L58 ?

Reply:
in order to have smartContractResults =  nil if len(sscr.Scrs) == 0